src/TelemetryConsumers/Orleans.TelemetryConsumers.Counters/OrleansPerformanceCounterInstaller.cs

```diff
@@ -1,94 +0,0 @@
-﻿using Orleans.Runtime;
-using System;
-using System.ComponentModel;
-using System.Configuration.Install;
-using System.Diagnostics;
-using System.Collections;
-using System.Collections.Generic;
-using System.Linq;
-using Microsoft.Extensions.Logging;
-using Orleans;
-using Orleans.ApplicationParts;
-using Orleans.Hosting;
-using Orleans.Logging;
-using Orleans.Metadata;
-using Orleans.Runtime.Configuration;
-
-namespace OrleansTelemetryConsumers.Counters
-{
-    /// <summary>
-    /// Providers installer hooks for registering Orleans custom performance counters.
-    /// </summary>
-    [RunInstaller(true)]
-    public class OrleansPerformanceCounterInstaller : Installer
```

Comment:
why is this removed? This installer will be invoked if the assembly is installed in Visual Studio Custom Action Installer or the Installutil.exe, to my understanding.

Reply:
Because there is no official nuget for `System.Configuration.Install`, as far as I know.

User can still use `OrleansCounterControl` to install them using a .exe